sinonjs/referee-sinon
# Fix browser compat

In the same way as https://github.com/sinonjs/referee/pull/216, this PR fixes browser compatibility by bundling node's `util.inspect`.

Bonus: switch to using `esbuild` for faster builds
Bonus bonus: remove `npm-run-all`


#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm run build`
4. grep the built file for `function inspect` to see that the `inspect` function actually got bundled

#### Checklist for author
